airflow/models/dagrun.py

```diff
@@ -1687,7 +1687,7 @@ class DagRunNote(Base):
 
     __tablename__ = "dag_run_note"
 
-    user_id = Column(Integer, nullable=True)
+    user_id = Column(String(100), nullable=True)
```

Comment:
Why do you make the user a lenght of 100 and the DB column in migration length of 128?

Reply:
Good catch! I'll fix that

Reply:
Fixed